docs/en/react/code-splitting.md

```diff
@@ -87,6 +87,12 @@ export function App() {
 }
 ```
 
+:::warning Differences from the Web
+In Lynx, CSS is scoped to each [Bundle](../api/lynx-native-api/template-bundle.md). Since lazy components generate their own Bundles (see output structure at [Output Files](../rspeedy/output.md)), you shouldn’t expect them to behave like on the Web. Global CSS defined in the main Bundle will not affect lazy components, and vice versa.
```

Comment:
Ideally speaking, there should be only "lazy-loaded components" and "`lazy ` component". 

You may want to update this and other occurrences of "lazy components" in this page

Reply:
Updated. Thanks! (Other occurrences in this page are all lazy-loaded components, no need to change.)